bindata/v3.11.0/openshift-apiserver/pdb.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: openshift-apiserver-pdb
+  namespace: openshift-apiserver
+spec:
+  maxUnavailable: 1
+  selector:
+    matchLabels:
+      app: openshift-apiserver-a
```

Comment:
okay, it matches https://github.com/openshift/cluster-openshift-apiserver-operator/blob/master/bindata/v3.11.0/openshift-apiserver/deploy.yaml#L33